src/main/java/com/intuit/graphql/orchestrator/utils/XtextTypeUtils.java

```diff
@@ -61,6 +77,18 @@ public static String getParentTypeName(FieldDefinition fieldDefinition) {
     }
   }
 
+  public static String getNamedTypeName(NamedType namedType) {
```

Comment:
com.intuit.graphql.utils.XtextTypeUtils.typeName(namedType)

Reply:
We should probably move some of the utility methods to that graphql-xtext repo

Reply:
https://github.com/intuit/graphql-xtext/blob/master/com.intuit.graphql.parent/com.intuit.graphql/src/com/intuit/graphql/utils/XtextTypeUtils.java